src/Categories/Category/Extensive.agda

```diff
@@ -32,13 +35,56 @@ record Extensive {o ℓ e : Level} (𝒞 : Category o ℓ e) : Set (suc (o ⊔ 
 
     disjoint : ∀ {A B : Obj} → IsPullback 𝒞 ¡ ¡ (i₁ {A = A}{B = B}) i₂
 
-
-
-
-
-
-
-
-
-
+  -- a version with non-canonical pullbacks
```

Comment:
I think you can make the following shorter by instead using a local module, i.e.

```agda
module _ {A B C : Obj} {f : A ⇒ B + C} (pb₁ : Pullback 𝒞 f i₁) (pb₂ : Pullback 𝒞 f i₂) where
```
and then do a few common `open` at the top.  You could also share some definitions in a `private` block that way.

Reply:
Indeed, thanks! Done the `module _` thing. Not sure about the `private` part.

Reply:
Did you mean to make the declarations before `pullback-of-cp-is-cp'` private to the module?